contracts/CO30_Jonty_GhostProtocol.zargabad/edit_me/briefing/mission.html

```diff
@@ -0,0 +1,5 @@
+NATO forces are facing a delicate situation in Zargabad. Recent clashes pushed back Taliban forces, but Mullah Rahmat, the regional leader, remains elusive. Local sympathies lie heavily with the Taliban, hindering NATO's ability to openly search for him.<br><br>
+NATO's embedded agent, Alex Adams, initially provided valuable intel on Taliban weapon caches and a key figure named Abu Ahmed operating out of Hazar Bagh. He is suspected of managing weapon caches delivered to the Taliban outpost in Hazar Bagh and using these weapons to mobilize new recruits. However, upon returning to Zargabad to deliver intel on potential checkpoint locations and Mullah Rahmat possibly hiding in the Abandoned Mosque, Adams went queit, possibly captured.<br><br>
+Synixe will infiltrate Zargabad disguised as Taliban fighters. Our primary objective is to locate and capture Abu Ahmed in Hazar Bagh and Mullah Rahmat in Zargabad, disrupting Taliban weapon stockpiles and recruitment efforts. Additionally locate and potentially rescue Alex Adams, leveraging intel he might have provided before capture. Destroy the ammo caches in Hazar Bagh to further cripple Taliban operations.<br><br>
```

Comment:
```suggestion
Our primary objective is to locate and capture Abu Ahmed in Hazar Bagh and Mullah Rahmat in Zargabad, disrupting Taliban weapon stockpiles and recruitment efforts. Additionally locate and potentially rescue Alex Adams, leveraging intel he might have provided before capture. Destroy the ammo caches in Hazar Bagh to further cripple Taliban operations.<br><br>
```

Your mission can not require everyone to buy new clothes